src/main/java/alfio/model/modification/PromoCodeDiscountModification.java

```diff
@@ -99,4 +99,60 @@ public int getDiscountValue(String currencyCode) {
         }
         return getDiscountInCents(currencyCode);
     }
+
+    public Integer getOrganizationId() {
```

Comment:
No need for the getters, they are auto generated by Lombok. You can remove them.